src/main/java/controller/LottoController.java

```diff
@@ -0,0 +1,61 @@
+package controller;
+
+import domain.*;
+import view.InputView;
+import view.OutputView;
+
+import java.util.List;
+import java.util.Random;
+
+public class LottoController {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    private int getLottoMoney;
+
+    public LottoController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void startLotto() {
+        getLottoMoney();
+        Lottos lottos = buyLotto();
+        List<Integer> lastWeekLottoNumber = getLastWeekLottoNumber();
+        rankLotto(lottos, lastWeekLottoNumber);
+    }
+
+    private void getLottoMoney() {
+        outputView.printGetLottoMoney();
+        getLottoMoney = inputView.getLottoMoney();
+    }
```

Comment:
```suggestion
    private int getLottoMoney() {
        outputView.printGetLottoMoney();
        return inputView.getLottoMoney();
    }
```

getLottoMoney를 메서드 지역 필드로 빼도 될 것 같아요~

Reply:
리뷰를 참고하여 리펙터링 완료하였습니다!